fe/fe-core/src/main/java/com/starrocks/authentication/AuthenticationManager.java

```diff
@@ -157,4 +168,93 @@ private void updateUserNoLock(
         }
         userToAuthenticationInfo.put(userIdentity, info);
     }
-}
+
+    /**
+     * Use new image format by SRMetaBlockWriter/SRMetaBlockReader
+     *
+     * +------------------+
+     * |     header       |
+     * +------------------+
+     * |                  |
+     * |  Authentication  |
+     * |     Manager      |
+     * |                  |
+     * +------------------+
+     * |     numUser      |
+     * +------------------+
+     * | User Identify 1  |
+     * +------------------+
+     * |      User        |
+     * |  Authentication  |
+     * |     Info 1       |
+     * +------------------+
+     * | User Identify 2  |
+     * +------------------+
+     * |      User        |
+     * |  Authentication  |
+     * |     Info 2       |
+     * +------------------+
+     * |       ...        |
+     * +------------------+
+     * |      footer      |
+     * +------------------+
+     */
+    public void save(DataOutputStream dos) throws IOException {
+        try {
+            // 1 json for myself,1 json for number of users, 2 json for each user(kv)
+            final int cnt = 1 + 1 + userToAuthenticationInfo.size() * 2;
```

Comment:
```suggestion
            final int cnt = 2 + userToAuthenticationInfo.size() * 2;
```

Reply:
The number here can be very confusing, that's why I added comment for this count. The original code matches the comment, making it easier to understand. I prefer to stay that way.